test/test_variational.py

```diff
@@ -291,6 +291,14 @@ def test_attrs(self):
         with self.assertRaisesRegex(AttributeError, 'Unknown variable name:'):
             dummy = fit.c
 
+    def test_timeout(self):
+        stan = os.path.join(DATAFILES_PATH, 'timeout.stan')
+        timeout_model = CmdStanModel(stan_file=stan)
+        self.assertRaisesRegex(
```

Comment:
Would you mind reorganizing these tests so they use the `with self.assertRaises...` style?